src/pages/internet-identity.tsx

```diff
@@ -63,355 +43,643 @@ function InternetIdentityPage() {
           marginTop: `calc(var(--ifm-navbar-height) * -1)`,
         }}
       >
-        <section
-          className="overflow-hidden bg-infinite text-white "
-          ref={heroRef}
-        >
+        <section className="bg-infinite text-white pt-20" ref={heroRef}>
           {bgDark && <DarkHeroStyles bgColor="transparent"></DarkHeroStyles>}
-          <AnimateSpawn
-            className="container-10 pt-40 md:pb-36 md:pt-64 relative"
-            variants={transitions.container}
-          >
-            <div className="md:w-7/10">
-              <motion.h1
-                className="tw-heading-3 md:tw-heading-2 mb-2 md:mb-6"
-                variants={transitions.item}
-              >
-                Internet Identity
-              </motion.h1>
+
+          <div className="container-10 pt-20 pb-12 sm:pb-0 md:pb-40 md:pt-36 relative">
+            <div className="blob blob-white blob-xl md:blob-xl md:blob-x-8 md:blob-y-10 opacity-100"></div>
+            <motion.h1
+              className="tw-heading-3 md:tw-heading-2 mb-2 md:mb-6 md:w-8/10"
+              variants={transitions.item}
+            >
+              Unlock the full potential of digital identity
+            </motion.h1>
+            <div className="relative  md:w-5/10">
               <motion.p
-                className="tw-lead-sm md:tw-lead mb-0"
+                className="tw-lead-sm md:tw-lead mb-8"
                 variants={transitions.item}
               >
-                <b>
-                Web 2.0 relies on usernames and passwords for authentication, in a paradigm that 
-                is insecure, inconvenient and fails to protect privacy.
-                </b>
-                <span style={{display: "block", height: "0.6em"}}></span>
-                Web3 often builds on centralized Web 2.0 infrastructure, and keeps associated tokens and NFTs 
-                on blockchains, which are controlled using cumbersome cryptographic wallets like MetaMask.
-                <span style={{display: "block", height: "0.6em"}}></span>
-                Internet Identity is an authentication system for both Web3 amalgams and full Web 3.0 combining public
-                standards such as WebAuthn and FIDO with chain key crypto technology &mdash; users conveniently create secure
-                sessions using fingerprint sensors and Face ID systems on their personal computing devices, or 
-                using external hardware wallets.
+                Internet Identity redefines user experiences by removing
+                friction from the authentication journey and enabling data
+                sovereignty.
+              </motion.p>
+              <motion.p className="mb-0" variants={transitions.item}>
+                <Link className="link-white link-with-icon" href="">
```

Comment:
does this need to route to another page?

Reply:
Thanks for the review @dfx-json , this is a work in progress, was meant to be a draft PR, as the copy is not final.